builtin/credential/chef-node/backend.go

```diff
@@ -0,0 +1,44 @@
+package chefNode
```

Comment:
That sounds fine, but make it `chefnode` please. Mixed case package names are uncommon.